Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -216,6 +216,15 @@ class PruneCoreSymbolicationdCacheIfTooLarge(shell.ShellCommand):
                "/System/Library/Caches/com.apple.coresymbolicationd"]
 
 
+class PruneCoreSymbolicationdDataIfTooLarge(shell.ShellCommand):
```

Comment:
This PR is just adding the step. Need to update factory as well to invoke this step.

Reply:
Indeed. I opened the PR early to get feedback from the Python linter. Related, I implemented the most naive solution to this problem. I'm not sure how to best go about parameterising `PruneCoreSymbolicationdCacheIfTooLarge` within our use of buildbot.